modules/training/ajax/markQuiz.php

```diff
@@ -29,41 +29,33 @@
 
 $quizCorrect = markQuiz($instrumentID);
 
-if ($quizCorrect == false) {
-    print "incorrect";
-    exit();
-} else {
-    $user = User::singleton();
-
-    $userFullName = $user->getFullname();
-    $userCenter   = $user->getCenterID();
-    $examinerID   = $DB->pselectOne(
-        "SELECT examinerID 
-         FROM examiners
-         WHERE full_name=:FN AND centerID=:CID",
-        array(
-         'FN'  => $userFullName,
-         'CID' => $userCenter,
-        )
-    );
+$userFullName = $user->getFullname();
+$userCenter   = $user->getCenterID();
+$examinerID   = $DB->pselectOne(
+    "SELECT examinerID 
+     FROM examiners
+     WHERE full_name=:FN AND centerID=:CID",
+    array(
+     'FN'  => $userFullName,
+     'CID' => $userCenter,
+    )
+);
 
-    $dateArray = array(
-                  'Y' => date('Y'),
-                  'M' => date('m'),
-                  'd' => date('d'),
-                 );
+$dateArray = array(
+              'Y' => date('Y'),
+              'M' => date('m'),
+              'd' => date('d'),
+             );
 
-    $values = array(
-               'pass'      => array($instrumentID => 'certified'),
-               'date_cert' => array($instrumentID => $dateArray),
-               'examiner'  => $examinerID,
-              );
+$values = array(
+           'pass'      => array($instrumentID => 'certified'),
+           'date_cert' => array($instrumentID => $dateArray),
+           'examiner'  => $examinerID,
+          );
 
-    process($values);
-
-    print "correct";
-    exit();
-}
+//process($values);
```

Comment:
@taracampbell This shouldn't be commented out I believe. When process is not called the completed certifications are not moved from "Certifications to complete" to "Completed Certifications"